packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountWaitScreen.js

```diff
@@ -1,89 +1,56 @@
 // @flow
 
 import * as React from 'react'
-import { ActivityIndicator, Text, View } from 'react-native'
+import { ActivityIndicator, View } from 'react-native'
+import { cacheStyles } from 'react-native-patina'
 
 import s from '../../../common/locales/strings.js'
 import * as Colors from '../../../constants/Colors.js'
-import * as Constants from '../../../constants/index.js'
-import * as Styles from '../../../styles/index.js'
-import { scale } from '../../../util/scaling.js'
-import { Header } from '../../common/Header.js'
-import SafeAreaView from '../../common/SafeAreaViewGradient.js'
+import {
+  type Theme,
+  type ThemeProps,
+  withTheme
+} from '../../services/ThemeContext'
+import { EdgeText } from '../../themed/EdgeText'
+import { ThemedScene } from '../../themed/ThemedScene'
 
 type Props = {}
 
-export class NewAccountWaitScreen extends React.Component<Props> {
-  render() {
-    return (
-      <SafeAreaView>
-        <View style={styles.container}>
-          <Header />
-          <View style={styles.pageContainer}>
-            <View style={styles.topPad} />
-            <View style={styles.iconContianer}>
-              <ActivityIndicator color={Colors.ACCENT_MINT} size="large" />
-            </View>
-            <View style={styles.headlineConainer}>
-              <Text style={styles.headlineText}>{s.strings.good_job}</Text>
-            </View>
-            <View style={styles.bodyCopyContainer}>
-              <Text style={styles.bodyText}>{s.strings.hang_tight}</Text>
-              <Text style={styles.bodyText}>{s.strings.secure_account}</Text>
-            </View>
-            <View style={styles.encriptingContainer}>
-              <Text style={styles.bodyText}>{s.strings.encrypting_wallet}</Text>
-            </View>
-          </View>
-        </View>
-      </SafeAreaView>
-    )
-  }
+const NewAccountWaitScreenComponent = ({ theme }: Props & ThemeProps) => {
```

Comment:
We already have a similar screen in the GUI: https://github.com/EdgeApp/edge-react-gui/blob/develop/src/components/scenes/CryptoExchangeQuoteProcessingScene.js

So instead of what you have or copying the GUI one, I want to write a component "WaitScreen" which accepts 2 props:

header: string | React.component
body: string | React.component